proto/bootz.proto

```diff
@@ -57,6 +86,54 @@ message GetBootstrapDataRequest {
   // Not settting this field indicates that the device is configured not to
   // check for ownership voucher.
   string nonce = 1001;
+
+  // Identity will contain information used to validate the identity
+  // of the device.
+  // For systems which support iDevID that will be the preferred 
+  Identity identity = 1003;
+} 
+
+// BootstrapStreamRequest is used to both build send chassis information
+// as well as build a trust relationship if the iDevID cert is not used
+// during TLS setup.
+message BootstrapStreamRequest {
+  message Response {
+    oneof type {
+      // Nonce to be signed with IAK from the device for iDevID based systems.
+      bytes nonce_signed = 1;
+      // For non-iDevID systems the encrypted_nonce will be decrypted via:
+      //   TPM 1.2: EK
+      //   TPM 2.0: PPK
+      string nonce = 2;
```

Comment:
Should we leave this as bytes ?

Reply:
since the getbootstraprequest contains a nonce  as a string i think we should keep the types consistent